_sips/process-specification.md

```diff
@@ -216,16 +221,16 @@ The SIP is now a recommendation for the compiler team or any other individual or
 group of people to provide an implementation of the proposal, as a pull request
 to the Scala 3 compiler repository. There is no set timeline for this phase.
 
-An implementation will be reviewed by the compiler team, and once the
-implementation is deemed good enough, it can ship as an Experimental feature in
-the following practical release of the compiler.
-
 Often, proposals not only need to be implemented in the compiler, but also in
 several other tools (IDEs, syntax highlighters, code formatters, etc.). As soon
-as a proposal reaches the implementation stage, the Committee notifies the
+as a proposal reaches the implementation stage, the Chairperson notifies the
 impacted tools that they should start implementing support for it. A list of
 tools of the ecosystem is maintained in [this document][tooling ecosystem].
 
+An implementation will be reviewed by the compiler team, and once the
+implementation is deemed good enough, it can ship as an Experimental feature in
+the following practical release of the compiler. At that point, the SIP Manager posts a follow-up comment on the Pre-SIP discussion to invite the community to test the feature and provide feedback.
```

Comment:
```suggestion
the next release of the compiler where it's practical to do so. At that point, the SIP Manager posts a follow-up comment on the Pre-SIP discussion to invite the community to test the feature and provide feedback.
```

Reply:
(guessing a bit about what you might mean here)

Reply:
I'd suggest s/practical/3.x minor/.

> it can ship as an Experimental feature in the following 3.x minor release of the compiler.

Scala 2 calls this a _major_ version, but 3 seems to call it a _minor_ version in [Long-term compatibility plans for Scala 3](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html). Maybe the word _practical_ got inserted in there to mean "next practically-possible release, like if a version is almost out the door, you don't have to shoehorn it in (you know what I mean)".

Reply:
Note that experimental features can be shipped in patch releases as well. I’ve applied Seth’s suggestion.